js/script.js

```diff
@@ -242,7 +242,10 @@ function commandMessage(key) {
 
         "windows,Python(3.5-3.7),X64,CUDA":
             "pip install onnxruntime-gpu",
-
+	
+	"linux,Python(3.5-3.7),X64,CUDA":
```

Comment:
change X64 to ARM64

Reply:
sorry, i think i need my afternoon coffee today. 3rd time's the charm. :P